actors/core/src/main/java/cloud/orbit/actors/extensions/NodeSelectorExtension.java

```diff
@@ -0,0 +1,14 @@
+package cloud.orbit.actors.extensions;
+
+import java.util.List;
+
+import cloud.orbit.actors.cluster.NodeAddress;
+import cloud.orbit.actors.extensions.ActorExtension;
+import cloud.orbit.actors.runtime.NodeInfo;
+
+public interface NodeSelectorExtension extends ActorExtension
+{
+
+    NodeInfo select(NodeAddress localAddress, List<NodeInfo> potentialNodes);
```

Comment:
interfaceClassName might be useful for providing custom node selector strategies based on actor, wdyt?